src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -198,101 +204,85 @@ export default class DashboardTable extends React.Component {
     formatRow(item, index) {
         let start = "Start: ";
         let end = "\nEnd: ";
-        if(this.state.type == 'fabs'){
-            start="Earliest: ";
-            end="\nLatest: "
+        if (this.state.type == 'fabs') {
+            start = "Earliest: ";
+            end = "\nLatest: "
         }
-        let reportingDateString = start+item.reporting_start_date + end + item.reporting_end_date;
+        let reportingDateString = start + item.reporting_start_date + end + item.reporting_end_date;
         if (!item.reporting_start_date || !item.reporting_end_date) {
             reportingDateString = 'No reporting period specified';
         }
 
-        let userName = '--';
-        if (item.hasOwnProperty('user')) {
-            userName = item.user.name;
-        }
+        let userName = item.hasOwnProperty('user') ? item.user.name : '--';
 
-        let deleteConfirm = false;
-        if(this.state.deleteIndex !== -1 && index === this.state.deleteIndex){
-            deleteConfirm = true;
-        }
+        let deleteConfirm = this.state.deleteIndex !== -1 && index === this.state.deleteIndex;
 
         let link = <SubmissionLink submissionId={item.submission_id} />;
-        if(this.state.type == "fabs") {
+        if (this.state.type == "fabs") {
             if(this.props.isCertified) {
                 link = <SubmissionLink submissionId={item.submission_id} value={reportingDateString} disabled={true}/>;
-            } else {
-                link = <SubmissionLink submissionId={item.submission_id} />;
             }
         } else if(this.props.isCertified) {
             link = <SubmissionLink submissionId={item.submission_id} value={reportingDateString} />;
         }
 
         let row = [];
-        if(this.state.type == 'fabs') {
-            if(this.props.isCertified){
-                let certified_on = item.certified_on;
-                if(certified_on !== "") {
-                    certified_on = this.convertToLocalDate(certified_on)
-                }
-                row = [
-                    link,
-                    item.agency,
-                    userName,
-                    this.convertToLocalDate(item.last_modified),
+        if (this.props.isCertified) {
+            // Certified Submissions table
+            row = [
+                link,
+                item.agency,
+                userName,
+                this.convertToLocalDate(item.last_modified)
+            ];
+
+            let certified_on = item.certified_on;
+            if (certified_on !== "") {
```

Comment:
why not do `certified_on = item.certified_on === "" ? "" : this.convertToLocalDate(item.certified_on)`